### PR TITLE
Proposed addition to unacceptable behavior section

### PR DESCRIPTION
**Pull request by ckolderup**
A collaborative effort between me and @eelzon (thanks Nozlee!), these are some edits we'd like to propose to the code of conduct as an addition to the "Unacceptable Behavior" section. It consists of:

1) adding `life experiences` to the list of attributes (line 7) about the people on our team, in part to encourage thinking about survivors of abuse/trauma, but also to recognize a major source of our differences
2) slightly restructuring the paragraphs (lines 9-13) to formally state our commitment to avoiding the below issues and make room for:
3) adding a section (line 15) about being intentional in language use to find appropriate terminology and metaphors for our work to be considerate of victims of state and individual violence

After the iterations visible in the commits below, this version has been approved by Cedric Chan on behalf of People & Culture and Lauren Fisher on behalf of the Legal team.

### CODE REVIEW

**Review by aworkinglibrary — APPROVED** [review]
Extremely good addition here, thank you!

angelfives

**Review by gesteves — APPROVED** [review]

**Review by fmanna — APPROVED** [review]
Excellent examples!

**Comment by banderson623** [comment]
First **these are great additions that I full support.**

Personally, I've learned so much, and continue to learn, how to respect others through the thoughtful and dignified writing of, and adherence to, these policies. 

This is not often rewarded or valued work. And doing this in the open and across teams is phenomenal – Thank you Cedric, @ckolderup, @eelzon,  Lauren and @aworkinglibrary for ensuring our company continues to improve our culture to make it more safe, inclusive and respectful.

**Review by annahsgraves — APPROVED** [review]

**Review by colettak — APPROVED** [review]

**Comment by sixfoot6** [comment]
💯

File: _posts/2015-10-08-unacceptable-behaviors.md [hunk 1]
```diff
@@ -4,8 +4,14 @@ title: Unacceptable behaviors
...
+The product team is committed to providing a welcoming and safe environment for people of all races, gender identities, gender expressions, sexual orientations, physical abilities, physical appearances, socioeconomic backgrounds, life experiences, nationalities, ages, religions, and beliefs. Discrimination and harassment are expressly prohibited in our employee handbook. Harassment includes, but is not limited to, intimidation; stalking; unwanted photography; inappropriate physical contact; use of sexual or discriminatory imagery, comments, or jokes; sexist, racist, ableist, or otherwise discriminatory or derogatory language; and unwelcome sexual attention.
```
Comment (livlab) [inline _posts/2015-10-08-unacceptable-behaviors.md:7]: Does it make sense to modify `unwanted photography` to `unwanted recording and photography` to explicitly include video and audio recording?

**Comment by livlab** [comment]
Nicely done all! Also props for the great references added. Small suggestion above.

File: _posts/2015-10-08-unacceptable-behaviors.md (hunk 1 above)
Comment (aworkinglibrary) [reply _posts/2015-10-08-unacceptable-behaviors.md:7]: No objections to this change, but it should be broken out into a separate PR.

Comment (ckolderup) [reply _posts/2015-10-08-unacceptable-behaviors.md:7]: I think it's a great suggestion and agree-- with such a minor change, I think we could probably get people to sign off on that much more quickly.

**Review by keaty — APPROVED** [review]

**Review by jamiemccarthy — APPROVED** [review]
These are great improvements, thank you.

**Review by ebinmore — APPROVED** [review]

**Review by stephenmckinney — APPROVED** [review]

**Pull request merged**